internal/agent/environment.go

```diff
@@ -125,7 +127,7 @@ func (a *Agent) CloseEnvironment(ctx context.Context, req *proto.CloseEnvRequest
 	vpnIP := strings.ReplaceAll(envConf.VPNAddress, ".240.1/22", "")
 	vpnIPPool.ReleaseIP(vpnIP)
 
-	if err := vbox.RemoveEventFolder(string(envConf.Tag)); err != nil {
+	if err := virtual.RemoveEventFolder(string(envConf.Tag)); err != nil {
 		//do nothing
 	}
```

Comment:
```suggestion
virtual.RemoveEventFolder(string(envConf.Tag))
```
Or at least log the error :)